src/pages/ProhibitorySigns/components/MapLayers/WideRoads.test.tsx

```diff
@@ -40,7 +39,7 @@ describe('ProhibitorySignsWideRoads', () => {
     // complete the wizard
     await user.click(screen.getByText('Kaart bekijken', { selector: 'button' }))
 
-    await user.click(await screen.findByLabelText('Breed opgezette wegen'))
+    await screen.findAllByText(/uw invoer/i)
```

Comment:
Sanity check. We hoeven hier geen user.click meer te doen?
En deze change heeft geen invloed op:
`expect(screen.getByLabelText(/breed opgezette wegen/i)).toBeEnabled()`

Reply:
Ja, de checkbox is standaard al aangevinkt, maar dat was blijkbaar niet aangepast in de test.